File: src/polcao/test/simple.c
#include <mpi.h>
#include <stdio.h>

int main(int argc, char** argv)
{
  unsigned int rank;
  unsigned int size;
  int mpierr;

  mpierr = MPI_Init(&argc, &argv);

  MPI_Comm_rank(MPI_COMM_WORLD, &rank);
  MPI_Comm_size(MPI_COMM_WORLD, &size);

  printf("I'm %u out of %u\n", rank, size);

  MPI_Finalize();
  return 0;
}

File: src/polcao/test/simple.f90
program gaBlockCyclic

!  use MPI

  implicit none

!#include "mafdecls.fh"
!#include "global.fh"

  integer :: mpiRank, mpiSize, mpierr
  integer :: i,j

  mpiRank = 2
  mpiSize = 0
  mpierr = 0

!  call mpi_init(mpierr)
!  print *, "Init: ", mpierr
!  call MPI_COMM_RANK(MPI_COMM_WORLD, mpiRank, mpierr)
!  print *, "Rank: ", mpierr
!  call MPI_COMM_SIZE(MPI_COMM_WORLD, mpiSize, mpierr)
!  print *, "Size: ", mpierr
!
!
!  print *, "\n\nStuff: ", mpiRank, mpiSize
!
!  call mpi_finalize(mpierr)

  do i=1,4
    do j=1,4
      print *, i, j
    enddo
  enddo
end program
